metabase/metabase
# Don't default to object detail for single result questions

Previously, any query that had a single result would go directly to the object detail view for that single result, and would not, in fact, allow the user to see the table view for that result at all. With the moving of object detail's primary view to a modal - this made that modal inescapable :cry: 

This changes that behavior to always show a single result in a table and require a user to click into that result to see the object detail view.

To test:
1. browse the orders table
2. filter > order id = 1
3. see single result in table
4. you can click into and out of object detail modal for that record

Resolves #21873 

![Screenshot from 2022-04-21 15-00-32](https://user-images.githubusercontent.com/30528226/164551714-9b774a64-5e65-420f-8682-e57f1fb87380.png)
